plugins/woocommerce-blocks/tests/e2e/bin/themes/twentytwentyfour-child__block-notices-filter/functions.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+add_filter( 'woocommerce_use_block_notices_in_classic_theme', '__return_false' );
```

Comment:
This filter is used to verify that block theme developers cannot trigger the classic notices via filter.